package.json

```diff
@@ -29,7 +29,8 @@
     "@types/stack-trace": "^0.0.29",
     "jest": "^27.5.1",
     "prettier": "^2.6.0",
-    "ts-jest": "^27.1.3"
+    "ts-jest": "^27.1.3",
+    "ts-node": "^10.7.0"
```

Comment:
Curious about this new dependency - are we using `ts-node` for something?

Reply:
Development tool for running typescript files directly.